### PR TITLE
Add timestamps to video 07: Thomas Wiecki part 2

### LINKED ISSUE #27: working on timestamps of video 07: Thomas Wiecki part2
Towards #11
Pair programming with Yuan Li 

In progress

### PR DESCRIPTION
**Pull request by AnaRita93**
### Reference
Towards #11
Closes #27  

### Description
Add timestamps for video 07: Thomas Wiecki Bayesian workflow part 2

### Event
#DataUmbrellaPyMCSprint

cc: @yyyyuans

### CODE REVIEW

File: videos-list/07-thomas_part2.md [hunk 1]
```diff
@@ -13,9 +13,41 @@ Discourse Discussion
...
+00:50 Agenda 
+01:08 Adressing Limitations of previous model using Rt
+01:31 Current Rt model in use from rt.live
+02:38 Delay effects on Rt we should take into account 
+03:06 Effects of Tracking lockdowns, mask use and social distancing on Rt
+
+03:45 The generative model 
+04:21 Dataset - US new daily cases 
+04:42 Build a generative model using a recursive function for new cases
+06:03 Mapp the recursive loop into pymc3 object with theano.scan
```
Comment (twiecki) [inline videos-list/07-thomas_part2.md:27]:
```suggestion
06:03 Translate the recursive loop into a pymc3 object with theano.scan
```

**Review by twiecki — APPROVED** [review]

**Comment by reshamas** [comment]
Thank you @AnaRita93 and @yyyyuans for the PR. 
Would you be able to:  
- make (commit) the updates as suggested by @twiecki 
- remove video 33, part 1, because that PR has already been accepted?

File: videos-list/07-thomas_part2.md [hunk 2]
```diff
@@ -13,9 +13,41 @@ Discourse Discussion
...
+23:18 Compare models 
+
+23:49 The number of infected - onset delay distribution
+24:50 Update model with onset delay effect
+26:26 Prior and Posterior predictive results 
+
+27:28 Adjusting for number of tests performed 
+29:35 Results discussion
+30:05 Conclusion/Final remarks
+
```
Comment (reshamas) [inline videos-list/07-thomas_part2.md:50]:
```suggestion
00:00 Intro - Motivation for Generative Models in Bayesian Statistics
00:50 Agenda 
01:08 Addressing Limitations of previous model using Rt
01:31 Current Rt model in use from rt.live
02:38 Delay effects on Rt we should take into account 
03:06 Effects of Tracking lockdowns, mask use and social distancing on Rt
03:45 The generative model 
04:21 Dataset - US new daily cases 
04:42 Build a generative model using a recursive function for new cases
06:03 Translate the recursive loop into a pymc object with theano.scan
07:06 Instantiate model and set parameters 
10:33 Follow the bayesian workflow 
11:25 Time varying reproduction
12:28 Select prior with time-varying factor (random walk process)
14:41 Update model with new prior
16:00 Prior and Posterior predictive results 
17:23 Taking infection delay into account - generation time distribution
18:00 Modeling of generation time distribution of covid 19 
19:03 Convolution definition
20:38 Updating generative process model with a convolution
21:50 Prior and Posterior predictive results  
23:18 Compare models 
23:49 The number of infected - onset delay distribution
24:50 Update model with onset delay effect
26:26 Prior and Posterior predictive results 
27:28 Adjusting for number of tests performed 
29:35 Results discussion
30:05 Conclusion / Final remarks
```

**Comment by reshamas** [comment]
Thank@ @AnaRita93 and @yyyyuans!
The timestamps have been added to the video.

**Comment by reshamas** [comment]
@AnaRita93 It looks like you may need to sync and your repo and push changes before I can merge?

**Comment by reshamas** [comment]
@OriolAbril
To discuss: how I can sync this PR to upstream and then merge it in.

**Pull request merged**